### PR TITLE
A complete management script.

### PR DESCRIPTION
**Pull request by h-alice**
In this PR, I merged the 10 dev-* scripts into a single file with subcommand strategy.
Every single script file has its corresponding subcommand:
- backend-debug: `dev-backend-debug.sh`
- shell: `dev-bash.sh`
- collect-static: `dev-collect-statics.sh`
- create-superuser: `dev-create-superuser.sh`
- keygen: `dev-keygen.sh`
- migration: `dev-migrations.sh`
- ipython: `dev-shell.sh`, I make different name to distinguish python shell from bash shell.
- ssh-shell: 'dev-ssh-bash.sh'
- django-startapp: `dev-startapp.sh`
- supervisorctl: `dev-supervisorctl.sh`

Still need help:
- [x] Check if the tooltip is correct. @NatLee 
- [x] Doublecheck if the merge cause unexpected behavior. @NatLee 
- [x] Remove the dev-* scripts if needed. @NatLee

### CODE REVIEW

**Comment by h-alice** [comment]
我思考了一個命名規則，大概是**常用的**名字短，不常用或**危險**的命名長避免誤爆。

**Comment by NatLee** [comment]
是說裡面其實有大量重複的`docker exec -it telepy-web-${PROJECT_NAME}`，應該可以拉出來做成一個constant？🤔

**Comment by h-alice** [comment]
> 是說裡面其實有大量重複的`docker exec -it telepy-web-${PROJECT_NAME}`，應該可以拉出來做成一個constant？🤔

可以欸 大概就跟docker的ARGS
然後一直引用差不多

**Review by NatLee — CHANGES_REQUESTED** [review]
We could call the script in the dev-scripts folder as a function. 🥹

**Comment by h-alice** [comment]
> We could call the script in the dev-scripts folder as a function. 🥹

OK! 👍

**Comment by NatLee** [comment]
`tooltip` looks OK.

```
Usage: ./telepy.sh sub-command [args]
Sub-commands:
  keygen: Generate SSH keys for Telepy service.
  create-superuser: Create a admin account for Telepy management.
  shell: Create a shell to run arbitrary command.
  ipython: Create a shell to run ipython.
  supervisorctl: Attach to supervisor control shell.
  ssh-shell: Similar to 'shell', but for ssh container.
  migration: Run migration process.
  backend-debug: Recreate and attach to backend container.
  collect-static: Collect static files to increase rendering speed.
  django-startapp: Create a new Django app.
```

But there's an error!

```bash
./telepy.sh supervisorctl
Error response from daemon: No such container: telepy-web-
```

I think we need to load `.env` with command `export $(grep -v '^#' .env | xargs)` in the top of the script.

**Review by NatLee — APPROVED** [review]
Seems OK! 👍🏼

File: telepy.sh
Comment (h-alice) [inline telepy.sh:1]: @NatLee 太神 竟然是 color terminal

Comment (NatLee) [reply telepy.sh:1]: uwu 必須

**Comment by h-alice** [comment]
Lets GO!

**Pull request merged**